docs/source/security/cors.mdx

```diff
@@ -84,108 +82,135 @@ If you create a public API or an API to embed in websites you don't control your
 
 #### None of the above
 
-If your application doesn't fit into any of the above categories, the default Apollo Server CORS behavior should suit your use case. You can always specify origins in your CORS configuration later or disable cross-origin requests by [disabling CORS for your server](../api/apollo-server#cors).
+If your application doesn't fit into any of the above categories, the default Apollo Server CORS behavior should suit your use case. You can always specify origins in your CORS configuration later or disable cross-origin requests by disabling CORS for your server.
+
+## Configuring CORS options for Apollo Server
 
-##  Configuring CORS options for Apollo Server
+> 📣 **New in Apollo Server 4**: if you are using an Apollo Server integration (e.g., `expressMiddleware`), you are responsible for [setting up CORS for your web framework](../migration#body-parser-and-cors).
```

Comment:
we'll want this link to not be to the migration guide eventually...

Reply:
I forgot to add a TODO there to add the API link later; I'll do that now across this doc thank you!